changes/3111.added

```diff
@@ -0,0 +1 @@
+Initial work on the controller model.
```

Comment:
```suggestion
Added `Controller` and `ControllerDeviceGroup` models.
```

Reply:
We also need a larger section in the `version-2.2.md` release notes highlighting this significant feature.